src/item_factory.cpp

```diff
@@ -2688,7 +2803,12 @@ void Item_factory::load_basic_info( const JsonObject &jo, itype &def, const std:
     if( jo.has_int( "to_hit" ) ) {
         assign( jo, "to_hit", def.m_to_hit, strict );
     } else if( jo.has_string( "to_hit" ) ) {
-        def.m_to_hit = parse_acc_string( jo.get_string( "to_hit" ) );
+        def.m_to_hit = acc_parse::parse_acc_string( jo.get_string( "to_hit" ) );
```

Comment:
in my personal opinion i would endeavor to obsolete the string way of writing to_hit in favor of object or integer.

Reply:
No option to encode via string; you and jbtw convinced me.